lib/templates/bootstrap/select.ios.js

```diff
@@ -23,6 +23,7 @@ class CollapsiblePickerIOS extends React.Component {
     let touchableStyleActive = stylesheet.pickerTouchable.active;
     let pickerValue = stylesheet.pickerValue.normal;
     if (locals.hasError) {
+      pickerContainer = stylesheet.pickerContainer.error;
```

Comment:
Seems that we forgot this. I'm not sure it would look great, but in the end its something everybody will customize very soon in their app development.